tests/auto_inject/test_auto_inject_install.py

```diff
@@ -61,6 +61,13 @@ def test_install(self):
         self._test_install(context.virtual_machine)
 
 
+@features.origin_detection
+@scenarios.container_auto_injection_install_script
+class TestContainerAutoInjectInstallScriptOriginDetection(base.AutoInjectBaseTest):
+    def test_install(self):
+        self._test_install(context.virtual_machine, origin_detection=True)
+
+
 @features.container_auto_installation_script_profiling
 @scenarios.container_auto_injection_install_script_profiling
 class TestContainerAutoInjectInstallScriptProfiling(base.AutoInjectBaseTest):
```

Comment:
nit: Do you think it's possible to add the check to [TestContainerAutoInjectInstallScript](https://github.com/DataDog/system-tests/blob/66e321c4a84b7d5ad0f81ad74140487252ad3da1/tests/auto_inject/test_auto_inject_install.py#L59) and [TestSimpleInstallerAutoInjectManua](https://github.com/DataDog/system-tests/blob/721d27176249150bfd18c6fe25c9a55a5847252c/tests/auto_inject/test_auto_inject_install.py#L173C7-L173C42)  to see if the other OOTB scenarios also have this issue?

Reply:
This PR is adding the checks to the `container_auto_injection_install_script` scenario but let me see what happens with the `simple_installer_auto_injection` & `multi_installer_auto_injection` scenarios.